lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -175,7 +202,16 @@ def braces_for_chaining_message(node)
           end
         end
 
+        def braces_required_message(node)
+          method_name = node.method_name.to_s
+          "Brace delimiters `{...}` required for '#{method_name}' method."
+        end
```

Comment:
We have a convention to use constants with format strings at the top of the class for messages:

```ruby
BRACES_REQUIRED_MESSAGE = "Brace delimiters `{...}` required for '%<method_name>s' method."

def braces_required_message(method_name)
  BRACES_REQUIRED_MESSAGE.format(method_name: method_name)
end
```

(Yes. It is ironic that we don't check this with RuboCop yet. 😉)

Reply:
Done in https://github.com/rubocop-hq/rubocop/pull/7660/commits/44c37b6fcf232cd7c401c80dbdc0bf41f36d1ad7 -- thanks!